CODE/_dev.py

```
@@ -1,33 +1,41 @@
-        with open(filename, "r+") as f:
-            data = json.load(f)
-            data[key] = new_data
-            f.seek(0)
+        config = configparser.ConfigParser(allow_no_value=True)
+        config.read(filename)
+        if key == "files":
+            config["System Settings"][key] = ", ".join(new_data)
+        elif key == "version":
+            config["System Settings"][key] = new_data
+        else:
+            log_dev.error(f"Invalid key: {key}")
```

Ruff (0.8.0)

Logging statement uses f-string

(G004)